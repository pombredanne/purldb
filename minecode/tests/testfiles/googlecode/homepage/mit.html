

<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"projectHomeUrl": "/p/annotare", "relativeBaseUrl": "", "assetHostPath": "http://www.gstatic.com/codesite/ph", "loggedInUserEmail": null, "profileUrl": null, "assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435", "domainName": null, "projectName": "annotare", "token": null};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 _gaq.push(
 ['projectTracker._setAccount', 'UA-1423305-7'],
 ['projectTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 annotare -
 
 
 a tool for annotating high-throughput biomedical investigations and resulting data - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .psicon-container {
 min-width:24px;
 }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Fannotare%2F&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Fannotare%2F" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/annotare">
 <a href="/p/annotare/">
 
 <img src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png" alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/annotare/"><span itemprop="name">annotare</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/annotare/"><span itemprop="description">a tool for annotating high-throughput biomedical investigations and resulting data</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/annotare/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/annotare/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/p/annotare/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/annotare/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/annotare/source/checkout"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 <span class="inst1">
 <a href="/p/annotare/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/annotare/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<table width="100%">
 <tr class="pscontent">
 <td class="pscolumnl">
 
 <div class="phead">Project Information</div>
 
 
 
 <ul class="pslist">
 
 <li class="psmeta">
 <a href="/p/annotare/feeds">Project feeds</a>
 </li>
 <li class="psgap"/>
 
 
 
 <li class="psmeta"><b>Code license</b></li>
 <li class="psmeta">
 <a href="http://www.opensource.org/licenses/mit-license.php" rel="nofollow">MIT License</a>
 
 </li>
 <li class="psgap"/>
 
 
 
 
 
 
 <li class="psmeta">
 <span id="project_labels">
 <b>Labels</b><br/>
 
 <a class="label"
 href="/hosting/search?q=label:bioinformatics">bioinformatics</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:microarray">microarray</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:annotation">annotation</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:mage-tab">mage-tab</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:mged">mged</a>
 
 </span>
 </li>
 <li class="psgap"/>
 
 
 </ul>
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_people-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Members</b></span>
 </div>
 <ul class="pslist">
 
 
 
 

 <a class="userlink"
 href="/u/105945310365770858596/">rdshan...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/105103350074067070456/">jun...@pcbi.upenn.edu</a
>, 
 
 

 <a class="userlink"
 href="/u/114026424382070757705/">AJCBurd...@gmail.com</a
>
 
 
 
 <li class="psmeta">
 <a href="people/list">6 committers</a>
 </li>
 
 
 <li class="psgap"/>
 
 
 </ul>
 
 
 <div class="phead pfeatured">Featured</div>
 
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_filedownload-g16 goog-inline-block vt"></div>
 </div>
 <span><b>Downloads</b></span>
 </div>
 <ul class="pslist">
 
 <li class="psmeta nowrap">
 <a title="Download" href="//annotare.googlecode.com/files/Annotare-v1.2.exe" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//annotare.googlecode.com/files/Annotare-v1.2.exe', 'Type-Installer, OpSys-Windows, Featured']);_gaq.push(['projectTracker._trackEvent', 'Download - Downloaded', '//annotare.googlecode.com/files/Annotare-v1.2.exe', 'Type-Installer, OpSys-Windows, Featured']);">
 </a>
 <a href="downloads/detail?name=Annotare-v1.2.exe"
 title="Annotare (release version 1.2) installer for Windows OS -- bundled with JRE and AIR runtime.">Annotare-v1.2.exe</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//annotare.googlecode.com/files/AnnotareManual.pdf" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//annotare.googlecode.com/files/AnnotareManual.pdf', 'Featured, Type-Docs']);_gaq.push(['projectTracker._trackEvent', 'Download - Downloaded', '//annotare.googlecode.com/files/AnnotareManual.pdf', 'Featured, Type-Docs']);">
 </a>
 <a href="downloads/detail?name=AnnotareManual.pdf"
 title="Annotare User Manual">AnnotareManual.pdf</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//annotare.googlecode.com/files/AnnotareTutorial.pdf" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//annotare.googlecode.com/files/AnnotareTutorial.pdf', 'Featured, Type-Docs']);_gaq.push(['projectTracker._trackEvent', 'Download - Downloaded', '//annotare.googlecode.com/files/AnnotareTutorial.pdf', 'Featured, Type-Docs']);">
 </a>
 <a href="downloads/detail?name=AnnotareTutorial.pdf"
 title="Annotare Tutorial">AnnotareTutorial.pdf</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//annotare.googlecode.com/files/Annotare_v1.3.pkg.zip" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//annotare.googlecode.com/files/Annotare_v1.3.pkg.zip', 'Featured, Type-Installer, OpSys-OSX']);_gaq.push(['projectTracker._trackEvent', 'Download - Downloaded', '//annotare.googlecode.com/files/Annotare_v1.3.pkg.zip', 'Featured, Type-Installer, OpSys-OSX']);">
 </a>
 <a href="downloads/detail?name=Annotare_v1.3.pkg.zip"
 title=" Annotare (release version 1.3) installer for Mac OS -- bundled with AIR runtime.  ">Annotare_v1.3.pkg.zip</a>
 </li>
 
 <li class="psmeta"><a href="downloads/list">Show all &raquo</a></li>
 <li class="psgap"/>
 </ul>
 
 
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_paper_pencil-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Wiki pages</b></span>
 </div>
 <ul class="pslist nowrap">
 
 <li class="psmeta">
 <a href="wiki/Release_Note" title=""
 >Release_Note</a>
 </li>
 
 <li class="psmeta"><a href="w/list">Show all &raquo</a></li>
 <li class="psgap"/>
 </ul>
 
 
 
 
 <div class="phead">Links</div>
 <ul class="pslist" style="white-space:nowrap">
 
 
 
 <li class="psmeta"><b>Groups</b></li>
 
 <li class="psmeta">
 <a href="http://groups.google.com/group/annotare-dev" rel="nofollow">Developers mailing list</a>
 </li>
 
 <li class="psmeta">
 <a href="http://groups.google.com/group/annotare-users" rel="nofollow">General users mailing list</a>
 </li>
 
 <li class="psgap"/>
 
 </ul>
 
 </td>
 <td id="wikicontent" class="psdescription">
 <h2><a name="Table_of_Contents"></a>Table of Contents<a href="#Table_of_Contents" class="section_anchor"></a></h2><p><ul><li><a href="#Introduction">Introduction</a></li><li><a href="#Release_Notes">Release Notes</a></li><li><a href="#Installation_Instructions">Installation Instructions</a></li><li><a href="#Documentation">Documentation</a></li><li><a href="#Support">Support</a></li><li><a href="#Acknowledgments">Acknowledgments</a></li><li><a href="#License">License</a></li></ul> </p><h1><a name="Introduction"></a>Introduction<a href="#Introduction" class="section_anchor"></a></h1><p>Annotare is a tool for annotating biomedical investigations and resulting data. It is a stand-alone desktop application that features 1) a set of intuitive editor forms to create and modify annotations, 2) support for easy incorporation of terms from biomedical ontologies, 3) standard templates for common experiment types, 4) a design wizard to help create a new document, and 5) a validator that checks for syntactic and semantic violation (see figure below). Annotare will help a bench biologist construct a MIAME-compliant annotation file based on the <a href="http://www.mged.org/mage-tab/spec1.0.html" rel="nofollow">MAGE-TAB</a> format. </p><blockquote><img src="http://annotare.googlecode.com/files/AnnotareComponents.jpg" /> 
</blockquote><h1><a name="Release_Notes"></a>Release Notes<a href="#Release_Notes" class="section_anchor"></a></h1><ul><li><a href="http://code.google.com/p/annotare/wiki/Release_Notes_1_3" rel="nofollow">http://code.google.com/p/annotare/wiki/Release_Notes_1_3</a> </li></ul><h1><a name="Installation_Instructions"></a>Installation Instructions<a href="#Installation_Instructions" class="section_anchor"></a></h1><h3><a name="Annotare_installers_are_available_for_Windows_(XP,Vista,_7)_and"></a>Annotare installers are available for Windows (XP,Vista, 7) and Mac (OS X) platforms<a href="#Annotare_installers_are_available_for_Windows_(XP,Vista,_7)_and" class="section_anchor"></a></h3><h4><a name="Requirements"></a>Requirements<a href="#Requirements" class="section_anchor"></a></h4><ol><li>Adobe AIR 1.5.2 or above  </li><li>Java 1.5 or above </li><li>At least 190M disk space for Mac and 70M for Windows </li></ol><p><strong>Downloads are subject to our <a href="http://www.opensource.org/licenses/mit-license.php" rel="nofollow">license agreement</a>.</strong> </p><h4><a name="Install_On_Windows_(XP,_Vista,_7)"></a>Install On Windows (XP, Vista, 7)<a href="#Install_On_Windows_(XP,_Vista,_7)" class="section_anchor"></a></h4><ol><li>Uninstall Annotare before installing a new version.  </li><li>Download the installer for <a href="http://annotare.googlecode.com/files/Annotare-v1.2.exe" rel="nofollow">Windows XP/Vista/2000/2003/2008 Installer bundled with JRE and AIR run time</a>      </li><li>The installer bundled with JRE will guide you through the Java installation process via a separate Java setup window. The installer will prompt you for confirmation to upgrade if it detects an older version of JRE. </li><li>The installer will add Annotare under the Programs menu. Go to Annotare under the Programs menu, and click on <i>startIt</i> to start Annotare. </li></ol><h4><a name="Install_on_Mac_(OS_X)"></a>Install on Mac (OS X)<a href="#Install_on_Mac_(OS_X)" class="section_anchor"></a></h4><ol><li>Use the Software Update feature (available from the Apple menu) to check that you have the latest version of Java.  </li><li>Before installing a new version, uninstall Annotare by removing it from /Applications folder. </li><li>Download the installer for <a href="http://annotare.googlecode.com/files/Annotare_v1.3.pkg.zip" rel="nofollow">Mac OS Installer bundled with AIR run time</a>.  </li><li>Double click on the zip file to unzip it to pkg file. Then double click on the pkg file to start Annotare installation process. </li><li>The installer will create Annotare.app under /Applications folder. Double click it to start Annotare. </li></ol><h4><a name="Uninstall_from_Windows_(XP,_Vista,_7)"></a>Uninstall from Windows (XP, Vista, 7)<a href="#Uninstall_from_Windows_(XP,_Vista,_7)" class="section_anchor"></a></h4><ol><li>Remove Annotare by using &#x27;Add/Remove programs&#x27; from control panel  </li><li>Delete the Annotare folder under the &#x27;Program Files&#x27; folder.  </li></ol><h4><a name="Uninstall_from_Mac_(OS_X)"></a>Uninstall from Mac (OS X)<a href="#Uninstall_from_Mac_(OS_X)" class="section_anchor"></a></h4><blockquote>Move Annotare.app from /Applications folder to Trash. 
</blockquote><h1><a name="Documentation"></a>Documentation<a href="#Documentation" class="section_anchor"></a></h1><ul><li><a href="http://annotare.googlecode.com/files/AnnotareManual.pdf" rel="nofollow">User Manual</a>  </li><li><a href="http://annotare.googlecode.com/files/AnnotareTutorial.pdf" rel="nofollow">Tutorial</a> </li><li><a href="http://annotare.googlecode.com/files/Annotare_Application_Note_Submitted.pdf" rel="nofollow">Publication: Annotare - a tool for annotating high-throughput biomedical investigations and resulting data, Bioinformatics. 2010</a> </li></ul><h1><a name="Support"></a>Support<a href="#Support" class="section_anchor"></a></h1><ul><li>Please email <a href="mailto:annotare-dev@googlegroups.com" rel="nofollow">annotare-dev@googlegroups.com</a> with questions, feature suggestions or comments.  </li><li>Please join the <a href="http://groups.google.com/group/annotare-users" rel="nofollow">users group</a> to receive announcements on Annotare, and also to post your questions and suggestions. </li></ul><p>  </p><h1><a name="Acknowledgments"></a>Acknowledgments<a href="#Acknowledgments" class="section_anchor"></a></h1><ul><li>Annotare development is supported by US National Institutes of Health (the National Human Genome Research Institute, the National Institute of Biomedical Imaging and Bioengineering and the National Cancer Institute) (grant number P41 HG003619), and EMBL. </li><li>Annotare is dependent on the following third-party software: </li><ol><li><a href="http://code.google.com/p/merapi/" rel="nofollow">Merapi project</a> </li><li><a href="http://sourceforge.net/projects/limpopo/" rel="nofollow">Limpopo project</a> </li></ol></ul><h1><a name="License"></a>License<a href="#License" class="section_anchor"></a></h1><p>Annotare is released under <a href="http://www.opensource.org/licenses/mit-license.php" rel="nofollow">MIT license</a> </p>
 </td>
 </tr>
</table>
<script src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/prettify/prettify_core_compiled.js"></script>
<script type="text/javascript">
 prettyPrint();
</script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

