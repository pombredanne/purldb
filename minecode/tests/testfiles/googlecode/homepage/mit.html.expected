{'content_license': None,
 'content_license_url': None,
 'description': u'a tool for annotating high-throughput biomedical investigations and resulting data',
 'downloads_url': u'http://code.google.com/p/annotare/downloads/list',
 'external_links': {},
 'feeds_url': u'http://code.google.com/p/annotare/feeds',
 'home_url': u'http://code.google.com/p/annotare/',
 'id': u'annotare',
 'issues_url': u'http://code.google.com/p/annotare/issues/list',
 'keywords': [],
 'labels': [u'annotation',
            u'bioinformatics',
            u'mage-tab',
            u'mged',
            u'microarray'],
 'license': u'MIT License',
 'license_url': u'http://www.opensource.org/licenses/mit-license.php',
 'name': u'annotare',
 'peoples_url': u'http://code.google.com/p/annotare/people/list',
 'vcs_url': u'http://code.google.com/p/annotare/source/checkout',
 'wiki_url': u'http://code.google.com/p/annotare/w/list'}